Data/DataLoader_mask2.py

```diff
@@ -23,41 +27,39 @@ def __init__(self, train_csv:str, val_csv:str, test_csv:str,batch_size:int , ima
         self.batch_size = batch_size
         self.image_column = image_column
         self.mask_column = mask_column
-        self.train_ds = None
-        self.val_ds = None
-        self.test_ds = None
-        # TO DO : Add the transforms (NORMALIZATION, RESIZE, TO TENSOR)
+        self.stage_ds = {'train': None, 'val': None, 'test': None}
+        # TO BE ADDED : Add the transforms (NORMALIZATION, RESIZE, TO TENSOR)
```

Comment:
I would appreciate if this is added before the merge since it's a crucial step in the training (we cannot start the training if we don't have the transforms)